content/using/develop.md

```diff
@@ -18,28 +22,42 @@ Once you’re ready, you can get started on the network right away and take some
 
 When developing, however, we recommend [booting a development ship](#creating-a-development-ship) and working locally -- you may need to make changes that could, in the wrong hands, destroy the ship’s environment, and it’s best to do so in a safe, separate place.
 
-## Core development {#core}
+## Applications on Urbit {#api}
 
-Working on the core means improving the Urbit project itself, working with the existing community of Urbit developers.
+Urbit ships with its own set of apps written in its own language, Hoon. However it is also meant to integrate with applications written in other languages via an API. The clearest example of an application for Urbit is Landscape, the default client. Although it does currently ship with Urbit, its front-end is a separate piece of software that uses the Urbit API to interact with an Urbit ship. back. You can create your own application that uses Urbit to store data and interact with other Urbit ships without writing any Hoon code thanks to this API. This could be a chat client, a writing app, a game, or anything else.
 
-Just arrived and unsure what to work on? An ideal way to get started is by experimenting with the system, talking to other developers, and reading (or [contributing to](https://github.com/urbit/docs)) the [documentation](/docs/).
+To get started quickly with the language of your choice, see the following libraries:
 
-Prefer learning with an instructor? We also offer an online course that covers the basics of Urbit development called [Hoon School](@/community/hoonschool.md). If course-based learning works well for you, we recommend you sign up.
+ - [Python](https://github.com/baudtack/urlock-py) (Stable)
```

Comment:
**suggestion:** adding a link to the Swift client as well would be a good idea: https://github.com/dclelland/UrsusAirlock